cli/packages/cmd/root.go

```diff
@@ -39,6 +39,7 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initLog)
 	rootCmd.PersistentFlags().StringP("log-level", "l", "info", "log level (trace, debug, info, warn, error, fatal)")
+	rootCmd.PersistentFlags().StringP("region", "r", "us", "Select the Infisical Cloud Region (us or eu). If the domain option is provided, this setting will be ignored.")
```

Comment:
I don't think we need a new flag. We can just tell them to set the `--domain` to be eu's address in the error message